src/layout/navbar/NavBar.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect } from 'react';
+import { Tooltip, UnstyledButton, Stack, rem } from '@mantine/core';
+import { IconHome2, IconGauge, IconFingerprint, IconLogout } from '@tabler/icons-react';
+import classes from './NavBar.module.css';
+import { RoutePaths } from 'Enum/Enum';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useAuth } from 'hooks/useAuth';
+import { persistor, useAppDispatch } from 'store/store';
+import { logout } from 'redux/slice/userSlice';
+interface NavbarLinkProps {
+  icon: typeof IconHome2;
+  label: string;
+  active?: boolean;
+  onClick?(): void;
+}
+
+function NavbarLink({ icon: Icon, label, active, onClick }: NavbarLinkProps) {
+  return (
+    <Tooltip label={label} position="right" transitionProps={{ duration: 0 }}>
+      <UnstyledButton
+        onClick={onClick}
+        className={`${classes.link} ${active ? classes.active : ''}`}
+        data-active={active || undefined}
```

Comment:
try to do it like this

```suggestion
        data-active={active}
```